nextjs-13/src/lib/showAlert/showAlert.ts

```diff
@@ -0,0 +1,12 @@
+import Swal, { SweetAlertIcon } from 'sweetalert2';
+import withReactContent from 'sweetalert2-react-content';
+
+const MySwal = withReactContent(Swal);
+
+export default async function makeSweetAlert(
```

Comment:
Did you miss changing the method name here or it is on purpose? 🤔 

Also in terms of directory structure, what do you think of:

```
├── lib
│   └── alert.ts
```

It looks repetitive to have `showAlert/showAlert.ts`.

Reply:
That's on purpose. Because I'm seeing the utils on `lib` are usually in folders on `lib`. Seems like the `alert.js` can be on `lib` without on another folder! Resolve [here](https://github.com/nimblehq/js-framework-benchmark/pull/116/commits/b8a09fa9e7118d734f0789147a374367b329cce6)!